extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -183,7 +217,111 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._dialogObject.content = [tab];
 	}
 
+	private createResouceButtonsContainer(): azdata.ListViewComponent {
+		const tags = this.getAllResourceTags(this._resourceTypes);
+		if (!tags.includes('All')) {
+			tags.splice(0, 0, 'All');
+		}
+		const items: azdata.ListViewOption[] = [];
+		tags.forEach((t: string) => {
```

Comment:
Just to make sure - we're ok with letting any arbitrary extension contribute categories? How are users supposed to know what categories exist? Might be better to start off with hardcoded list of categories initially...